common.mk

```diff
@@ -268,6 +268,11 @@ DEFCONFIG_XEN=--br-defconfig build/br-ext/configs/xen.conf
 # The version of Xen provided by Buildroot needs a few patches to work with
 # OP-TEE
 BR2_GLOBAL_PATCH_DIR=../build/br-ext/patches
+
+# See if we run Xen with FF-A
+ifneq ($(filter 1 2 3,$(SPMC_AT_EL)),)
+XEN_FFA=y
```

Comment:
one space on each side of `=`

Reply:
I'll remove this, keeping only the `qemu_v8.mk` part.